src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-create-modal/bootstrap-create-modal.component.html

```diff
@@ -69,8 +69,20 @@
                     i18n-placeholder
                     id="token"
                     formControlName="token"
-                    readonly>
-          </textarea>
+                    readonly
+                    #token></textarea>
+        </div>
+
+        <div class="btn-group mb-3"
+             role="group"
+             aria-label="Copy To Clipboard">
+          <button class="btn btn-primary"
+                  title="Copy to Clipboard"
+                  i18n-title
+                  (click)="copyToClipBoard(token)"> Copy to Clipboard
```

Comment:
```suggestion
                  (click)="copyToClipBoard(token)"
                  i18n>Copy to Clipboard
```

Reply:
I have added <ng-container i18n>Copy to Clipboard</ng-container> as per the review above